### DISCUSSION TITLE
Change header text on Kiosk screen?

### DISCUSSION THREAD

**Discussion by bokerlol** [kimai/kimai#3637] [Questions & Answers] [answered]
Is it possible to change the text on the kiosk screen from 'Please scan your employee working card' to something else? We aren't using badges so we have no use for the text and I don't want it to confuse anyone as we do have badges for entering our building.

**Comment by kevinpapst** [comment, marked as answer]
Yes, with the translation plugin, which allows to change most of all displayed text elements in Kimai :

<img width="804" alt="Bildschirmfoto 2022-11-17 um 20 30 48" src="https://user-images.githubusercontent.com/533162/202541008-24bffedb-40df-4de7-a7d7-81898db2fd9a.png">

<img width="1541" alt="Bildschirmfoto 2022-11-17 um 20 31 12" src="https://user-images.githubusercontent.com/533162/202541120-73be8bf1-1c40-45d3-834f-cea19b0ec559.png">

See https://www.kimai.org/store/translation-bundle.html

**Reply by bokerlol** [reply]
Buying it now! Thank you!!

**Reply by bokerlol** [reply]
I didn't see it when searching through the options in the translation plugin, but is it possible to change 'Code *' to something else?

**Reply by kevinpapst** [reply]
Weirdly enough, there was no translation for that label. I just added it. 

Please sent me an email from your company address and I will add you to the repositories for your purchased plugins, where you can download the most recent changes.

As explained here: https://www.kimai.org/documentation/buy-a-plugin.html

**Reply by bokerlol** [reply]
Just emailed in. Thank you!

**Reply by bokerlol** [reply]
Hey @kevinpapst, did you push the update to the TranslationBundle for the thing we mentioned above?
